ui/components/ui/button/buttons.scss

```diff
@@ -2,15 +2,15 @@
   Buttons
  */
 
-$hover-secondary: #b0d7f2;
-$hover-default: #b3b3b3;
-$hover-confirm: #0372c3;
-$hover-red: #feb6bf;
-$hover-red-primary: #c72837;
-$hover-orange: #ffd3b5;
-$warning-light-orange: #f8b588;
-
 .button {
+  --hover-secondary: #b0d7f2;
```

Comment:
Are these new colors or do they match any existing colors?

Reply:
They match the colors cited above.